platforms/hdp2.6.1/docs/HDP-installation.md

```diff
@@ -138,6 +155,98 @@ If you're using Kerberos, you need to add this option to your HBase Master Java
    ````
    -Dsplice.spark.hadoop.fs.hdfs.impl.disable.cache=true
    ````
+   
+### Enabling Ranger for Authorization
+
+Splice Machine installs with Native authorization configured; native
+authorization uses the Splice Machine dictionary tables to determine permissions on database objects.
+
+
+#### Config Splice Machine Ambari Service
+
+
+In the tab:  Advanced ranger-splicemachine-audit
+
+1. Check audit to HDFS
+2. Check audit to SOLR
+3. For the config: xasecure.audit.destination.solr.urls change localhost to the hostname / node 
+for SOLR
+4. Set xasecure.audit.is.enabled to true
+
+In the tab:  Advanced ranger-splicemachine-security
+
+1. Update ranger.plugin.splicemachine.policy.rest.url
+2. Change localhost to the host / node for Ranger Server
+
+
+#### Add Ranger Service for Splice Machine
+
+Before changing the authorization scheme, the Splice Machine ranger service needs to be installed.  As part of the Splice Machine Ambari Service, 
+the admin plugin for Splice Machine is added to the Ranger web application.
+
+The service can be installed by executing the following from a command line on the machine where the Ambari Service resides.
+
+Then post this file to Ranger API. Run the command bellow on master. `admin:admin` here is 
+Ranger's username and password.
+
+```
+curl -sS -u admin:admin -H "Content-Type: application/json" -X POST http://localhost:6080/service/plugins/definitions -d @/var/lib/ambari-server/resources/stacks/HDP/2.6/services/SPLICEMACHINE/configuration/ranger-servicedef-splicemachine.json
+```
+1. Go to Ranger admin web page.
+2. You should see SpliceMachine Plugin
+3. Click on the plus sign (+) next to SpliceMachine
+4. Need to add a Service: the service name is the same name as you configured in
+`ranger.plugin.splicemachine.service.name`, which is `splicemachine` by default.
+
+Note: if you see some error like this when click "test connection":
+
+```
+Unable to retrieve any files using given parameters, You can still save the repository and start
+creating policies, but you would not be able to use autocomplete for resource names.
+Check ranger_admin.log for more info.
+
+org.apache.ranger.plugin.client.HadoopException: Unable to login to Hadoop environment [splicemachine]. 
+Unable to login to Hadoop environment [splicemachine]. 
+Unable to decrypt password due to error. 
+Input length must be multiple of 8 when decrypting with padded cipher. 
+```
+
+It is because of a [Ranger bug](https://issues.apache.org/jira/browse/RANGER-1640?attachmentOrder=asc).
```

Comment:
this URL can probably be shortened to:
https://issues.apache.org/jira/browse/RANGER-1640